src/yamlprocessor/__init__.py

```diff
@@ -1,2 +1,2 @@
 """Process YAML include files and variable substitutions."""
-__version__ = '0.6'
+__version__ = '0.6.1'
```

Comment:
I want to create a release right after this, given the bug is a blocker for mo-bundle new stack.